### PR TITLE
:hammer_and_wrench: + :lipstick: |  fix + pretty + clean

### PR DESCRIPTION
**Pull request by miko-github**
# :hammer_and_wrench: + :lipstick: :=  fix + pretty + clean
- fix : string concats with literally string
- pretty : hard readable if condition with constant value
- fix :  hard readable constant with functionality checking

:v: :heart:

### CODE REVIEW

File: main/main.js [hunk 1]
```diff
@@ -21,21 +45,23 @@ crashReporter.start({
...
+// arg === '--development-mode' -> arg === CMD_DEV_MODE
 let isInstallerRunning = false
 const isDevelopmentMode = process.argv.some(arg => arg === '--development-mode')
 
 function clamp (n, min, max) {
   return Math.max(Math.min(n, max), min)
 }
 
-if (process.platform === 'win32') {
+if (process.platform === PLATFORM_WIN) {
```
Comment (lubiedo) [inline main/main.js:61]: Isn't `process.platform === PLATFORM_WIN` exactly the same as doing `process.platform === 'win32'` and similars?

File: main/main.js [hunk 2]
```diff
@@ -277,7 +303,7 @@ function createWindowWithBounds (bounds) {
   so registering a handler causes events to happen twice.
   See: https://github.com/electron/electron/issues/18322
   */
-  if (process.platform === 'win32') {
+  if (isPlatform('win32')) {
```
Comment (lubiedo) [inline main/main.js:306]: Why here are you using `isPlatform('win32')` but in other changes `process.platform === PLATFORM_WIN`?

File: main/main.js [hunk 3]
```diff
@@ -21,21 +45,23 @@ crashReporter.start({
...
 
 if (process.argv.some(arg => arg === '-v' || arg === '--version')) {
-  console.log('Min: ' + app.getVersion())
-  console.log('Chromium: ' + process.versions.chrome)
+  console.log(`Min: ${app.getVersion()}`)
+  console.log(`Chromium: ${process.versions.chrome}`)
   process.exit()
 }
 
+// arg === '--development-mode' -> arg === CMD_DEV_MODE
```
Comment (lubiedo) [inline main/main.js:53]: What's the purpose of this?

**Review by lubiedo — COMMENTED** [review]
Hi @miko-github. I'm kinda confused about these changes and left some reviews.

**Comment by miko-github** [comment]
> Hi @miko-github. I'm kinda confused about these changes and left some reviews.

# Hi @lubiedo 👋

Actually which line? tag them to me and let me to explain 😉🤗

File: main/main.js (hunk 1 above)
Comment (miko-github) [reply main/main.js:61]: Yes, but, constant variables be clean and human readable as like Enums  😉✌

File: main/main.js (hunk 2 above)
Comment (miko-github) [reply main/main.js:306]: I just commuted options 😉

I just say, we can use this very clear....

Use function or constant or enum 😁

File: main/main.js (hunk 3 above)
Comment (miko-github) [reply main/main.js:53]: Actually, constant variables again but in optional, and you can change name to anything else you want 🤗

But I'm suggesting you, using enum, I know enum isn't available in JS but we can make similar of them...

```js
const SHELL_FLAG = {
  DEVELOPER_MODE: '--developer-mode'
}
```
In usage this be like enum,
```js
SHELL_FLAG.DEVELOPER_MODE
```

**Comment by PalmerAL** [comment]
I’m also fairly confused by this. Putting things in constants doesn’t automatically make them more readable; constants can help at reducing duplication and making it easier to change values, but here nothing’s really being duplicated to start with. And you can’t even reuse the same value everywhere, because `navigator.platform` and `process.platform` use different platform names, and we use both.

**Comment by miko-github** [comment]
> I’m also fairly confused by this. Putting things in constants doesn’t automatically make them more readable; constants can help at reducing duplication and making it easier to change values, but here nothing’s really being duplicated to start with. And you can’t even reuse the same value everywhere, because `navigator.platform` and `process.platform` use different platform names, and we use both.

# Okay :+1: , now i understood :smile:, thanks :hugs: :heart:

**Pull request closed without merging**